en/intro_to_command_line/open_instructions.md

```diff
@@ -26,3 +26,12 @@ Go to Applications → Utilities → Terminal.
 It's probably under Applications → Accessories → Terminal, or Applications → System → Terminal, but that may depend on your system. If it's not there, you can try to Google it. :)
 
 <!--endsec-->
+
+<!--sec data-title="Opening: RunCode" data-id="runcode_prompt" data-collapse=true ces-->
```

Comment:
As previous comments - sub heading also needed for existing instructions above

Reply:
> As previous comments - sub heading also needed for existing instructions above

I have added instructions in the Installation Readme to give instructions that one should follow instructions for their platform and skip all others to avoid the repetition of adding that everywhere so this should now work as is with the new changes.